### PR TITLE
New In Chrome 96 post

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- New in Chrome 96 post

TODO Before Publish
- [x] Update YouTube video URL

### CODE REVIEW

File: site/en/blog/new-in-chrome-96/index.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+## Manifest `id` for PWAs {: #manifest-id }
+
+When a user installs a PWA, the browser needs a way to uniquely identify it.
+But, until recently, the web app manifest spec didn't specify how to identify
+a PWA, leaving browsers to decide, and leading to different implementations.
+In some browsers, the `start_url` is used, while in others, the path to the
+manifest file is used.
+
+That makes it impossible to change either of those fields without breaking the
+install experience! Now, there's a new optional `id` property, that allows
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:49]:
```suggestion
install experience. Now, there's a new optional `id` property, that allows
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  ...
+  id: "/?homescreen=1",
+  start_url: "/?homescreen=1",
+  ...
+}
+```
+
+Support for the `id` property lands in desktop Chromium-based browsers
+starting in Chrome 96. Support for mobile, which currently uses the manifest
+url as the unique id, should land in the first half of 2022.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:67]:
```suggestion
url as the unique ID, should land in the first half of 2022.
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+{% Img class="float-right", linkTo="true", src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/XY3FVUeeCjhB9K71niJE.png", alt="DevTools showing computed app id", width="800", height="533" %}
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:69]:
```suggestion
{% Img class="float-right", linkTo="true", src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/XY3FVUeeCjhB9K71niJE.png", alt="DevTools showing computed app ID", width="800", height="533" %}
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+If you already have a PWA in production and want to add an `id` to your
+manifest, you'll need to use the id that was assigned by the browser. You can
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:72]:
```suggestion
manifest, you'll need to use the ID that was assigned by the browser. You can
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+```
+
+Check out [Uniquely identifying PWAs with the web app manifest id property][manifest-id]
+for more info.
+
+## URL protocol handlers for PWAs {: #pwa-protocol-handlers }
+
+Web apps can use [`navigator.registerProtocolHandler()`][mdn-regph] to
+register as a protocol handler. For example, Gmail can register the `mailto`
+protocol, then when a user clicks on a link with the `mailto:` prefix, Gmail
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:96]:
```suggestion
protocol. Then when a user clicks on a link with the `mailto:` prefix, Gmail
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+will open making it easy for the user to send an email.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:97]:
```suggestion
will open, making it easy for the user to send an email.
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+Starting in Chrome 96, a PWA can register as protocol handler as part of its
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:99]:
```suggestion
Starting in Chrome 96, a PWA can register as a protocol handler as part of its
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+<!-- We don't want a high priority for this above-the-fold image -->
+<img src="/not-important.svg" importance="low">
+
+<!-- Initiate an early fetch for a resource, but de-prioritize it -->
+<link rel="preload" href="/script.js" as="script" importance="low">
+
+<script>
+  fetch('https://example.com/', {importance: 'high'}).then(data => {
+    // Trigger a high priority fetch
+  });
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:145]:
```suggestion
  fetch('https://example.com/', {importance: 'high'})
  .then(data => {
    // Trigger a high priority fetch
  });
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+Early next year, we'll hit Chrome 100, a **three** digit version number. Any
+code that checks version numbers, or parses the UA string, should be checked
+to make sure it handles three digits.
+
+Starting in Chrome 96, there's a new flag `#force-major-version-to-100` that
+will change the current version number to 100, so you can make sure
+everything works as expected.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Screenshot of Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:174]:
```suggestion
{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
```

File: site/en/blog/new-in-chrome-96/index.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+* The Back, forward cache – or `bfcache` – is now available in stable, and
+  brings Chrome in line with both Firefox and Safari.
+
+## Further reading
+
+This covers only some of the key highlights. Check the links below for
+additional changes in Chrome 96.
+
+* [What's new in Chrome DevTools (96)](/blog/new-in-devtools-96/)
+* [Chrome 96 deprecations & removals](/blog/deps-rems-96/)
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-96/index.md:198]:
```suggestion
* [Chrome 96 deprecations and removals](/blog/deps-rems-96/)
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/new-in-chrome-96/index.md (hunk 1 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+In some browsers, the `start_url` is used, while in others, the path to the
+manifest file is used.
+
+That makes it impossible to change either of those fields without breaking the
+install experience! Now, there's a new optional `id` property, that allows
+you to explicitly define the identifier used for your PWA.
+
+Adding the `id` property to the manifest removes the dependency on the
+`start_url` or the location of the manifest, and makes it possible for those
+fields to be updated!
```
[the remark this answers is not in the document]
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 2 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 3 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 4 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 8 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+to `high` on the background image, the LCP drops from 2.6 seconds to 1.9
+seconds.
+
+{% Video src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/MGM6YFi3IHrjRoguRr92.mp4", autoplay="true", muted="true", loop="true" %}
+
+Check out [Optimizing resource loading with Priority Hints][wd-phints] for
+all the details, how to register for the origin trial, and some great examples
+of how it can help to improve your rendering performance.
+
+## Emulate Chrome 100 in UA string {: #chrome-100 }
```
[the remark this answers is not in the document]
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 9 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+code that checks version numbers, or parses the UA string, should be checked
+to make sure it handles three digits.
+
+Starting in Chrome 96, there's a new flag `#force-major-version-to-100` that
+will change the current version number to 100, so you can make sure
+everything works as expected.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Screenshot of Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
+
+For more details see [Force Chrome major version to 100 in the User-Agent string][dcc-cr100].
```
[the remark this answers is not in the document]
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

File: site/en/blog/new-in-chrome-96/index.md (hunk 10 above)
Comment (petele) [reply site/en/blog/new-in-chrome-96/index.md]: Fixed in 1475b8e031abe034773b744274e64066738153ad

**Review by jpmedley — APPROVED** [review]

**Pull request merged**